templates/patroni.yml.j2

```diff
@@ -63,12 +63,11 @@ bootstrap:
     {% endif %}
   {% endfor %}
 {% endif %}
-{% if patroni_bootstrap_pg_hba |d([], true) |length > 0 %}
   pg_hba:
-  {% for client in patroni_bootstrap_pg_hba %}
-    - {{ client.type }} {{ client.database }} {{ client.user }} {{ client.address|d(None) }} {{ client.method }}
+  {% for host in groups['patroni'] %}

```

Comment:
Currently, I don't have an optimal decision regarding this section, but definitely this part should be written in other way. Let me think about it a little.